tests/integ/bootstrap.exp

```diff
@@ -18,7 +18,7 @@ spawn $ziti edge quickstart --home $env(QUICKSTART_HOME)
 set qs_id @spawn_id
 # wait until ER is connected
 expect {
-    "finished sychronizing api sessions" { }
```

Comment:
this typo is needed as it is the output from another project

Reply:
if you also wish to fix that -- it is here:
https://github.com/openziti/ziti/blob/42f6e3fc487b70970b4b6f57b0af72ebf06a65df/router/handler_edge_ctrl/apiSessionAdded.go#L124

Reply:
@ekoby  thank you. I´ve created a PR there, see https://github.com/openziti/ziti/pull/1652